database/migrations/2019_04_08_142523_create_exchange_rates_table.php

```diff
@@ -0,0 +1,47 @@
+<?php
+/**
+ * Copyright (c) 2018-2019 Adshares sp. z o.o.
+ *
+ * This file is part of AdServer
+ *
+ * AdServer is free software: you can redistribute and/or modify it
+ * under the terms of the GNU General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ *
+ * AdServer is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty
+ * of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with AdServer. If not, see <https://www.gnu.org/licenses/>
+ */
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+class CreateExchangeRatesTable extends Migration
+{
+    public function up(): void
+    {
+        Schema::create(
+            'exchange_rates',
+            function (Blueprint $table) {
+                $table->increments('id');
+                $table->timestamps();
+                $table->timestamp('valid_at')->nullable(false)->default('1970-01-01 00:00:01');
```

Comment:
```suggestion
                $table->timestamp('valid_at')->nullable(false)->unique();
```

Reply:
Solved by moving timestamps at the end of columns definitions